test/e2e/pages/population-weekly.js

```diff
@@ -0,0 +1,23 @@
+import { Selector, t } from 'testcafe'
+
+import Page from './page'
+
+class PopulationWeeklyPage extends Page {
+  constructor() {
+    super()
+
+    this.nodes = {
+      breadcrumbs: {
+        overview: Selector('.govuk-breadcrumbs__link'),
+      },
+    }
+  }
+
+  async visitOverviewPage() {
```

Comment:
This function isn't needed, and should be removed.

Reply:
Is this comment saying this it needs to be removed with this change or in future?

Reply:
It's a note to myself to remove it as part of this PR - updated the comment to make that more obvious :)